ci-operator/config/pulp/pulp-operator/OWNERS

```diff
@@ -0,0 +1,21 @@
+approvers:
+- chambridge
+- dkliban
+- fao89
+- git-hyagi
+- mdellweg
+- mikedep333
+- pulpbot
+- rooftopcellist
+- spredzy
```

Comment:
@chambridge
@dkliban
@git-hyagi
@mdellweg
@mikedep333
@rooftopcellist
@spredzy
could you all please comment: `/approve`on this PR?
It will enable OCP CI for pulp-operator

Reply:
@smg247 should I add you on the OWNERS file?
No one from my team is a collaborator on openshift org

Reply:
/approve

Reply:
No, don't add me there. You need to get org membership for your team by reaching out to DPP at #forum-dp-platform on slack.

Reply:
FYI you don't need _everyone_ to approve, just one person from each file. DPTP is able to approve _any_ file so this was already approved by me, but you do need to get org membership for yourself and your team.

Reply:
Thank you!

Reply:
@smg247 could you please lgtm again?